docs/asset-store/11-01-asset-store-controller.md

```diff
@@ -8,4 +8,14 @@ Metrics for the Asset Store Controller include:
 - default metrics instrumented by [kubebuilder](https://book.kubebuilder.io/).
 - default Prometheus metrics for [Go applications](https://prometheus.io/docs/guides/go-application/).
 
+To see a complete list of the default kubebuilder and Go metrics, run the following command:
+
+```bash
+kubectl -n kyma-system port-forward svc/assetstore-asset-store-controller-manager 8080
+```
+
+Now open a browser and access [http://localhost:8080/metrics](http://localhost:8080/metrics) to check the metrics.
```

Comment:
not browser, please do dev instruction like `curl http://localhost:8080/metrics` from the terminal

so you can write:
```bash
kubectl -n kyma-system port-forward svc/assetstore-asset-store-controller-manager 8080
curl http://localhost:8080/metrics
```

you can copy/paste above and should work straight forward in the terminal

Reply:
I probably I do something wrong as the terminal does not read the curl command once I past it as you advised. 

If it works, then we could do the redirecting in the same way, I assume?
```
kubectl -n kyma-system port-forward svc/assetstore-asset-store-controller-manager 3000:8080
curl http://localhost:3000/metrics
```
?

Reply:
As we discussed the curl command should be run in another terminal window.